### PR TITLE
Bump SDK version

### PR DESCRIPTION
**Pull request by utpilla**
- Similar to https://github.com/open-telemetry/opentelemetry-dotnet/pull/4759
- Simplify dotnet-setup in CI workflows

### CODE REVIEW

**Comment by CodeBlanch** [comment]
@utpilla FYI it looks like a lot of the CI in this repo passes a channel to `setup-dotnet` for example:

https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/b0e0828e015293326051eff7fbe1515155ba44c5/.github/workflows/ci.yml#L31-L32

I think we could remove that and it will just defer to whatever is defined in global.json. Less junk to modify anytime we bump the SDK that way 😄

**Review by vishweshbankwar — APPROVED** [review]

**Review by Yun-Ting — APPROVED** [review]

**Review by alanwest — APPROVED** [review]
✅

File: .github/workflows/package-Instrumentation.StackExchangeRedis.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Pack OpenTelemetry.Instrumentation.StackExchangeRedis
+Profilername: Pack OpenTelemetry.Instrumentation.StackExchangeRedis
```
Comment (CodeBlanch) [inline .github/workflows/package-Instrumentation.StackExchangeRedis.yml:1]: @utpilla What's up with this "Profiler" bit? Copy/paste fail?

Comment (utpilla) [reply .github/workflows/package-Instrumentation.StackExchangeRedis.yml:1]: Great catch! And yes looks like a terrible copy/paste fail 😂

**Pull request merged**